src/components/molecules/ChatMessageBox/ChatMessageBox.scss

```diff
@@ -1,5 +1,7 @@
 @import "scss/constants";
 
+$emoji-button-diameter: $submit-button-diameter - $spacing--md;
```

Comment:
Due to expediency of PR a more holistic solution wasn't pursued i.e. actually making the emoji button part of the input component, possibly creating a new input component. It is OOS to this PR, but an idea for further consideration.